src/core/pages/PatientSummary/index.js

```diff
@@ -37,25 +36,29 @@ class PatientSummaryInfo extends Component {
     }
 
     render() {
-        const { classes, patientInfo, loading } = this.props;
-        const coreSynopsisData = getSynopsisData(patientInfo);
+        const { classes, patientInfo, loading, showMode, showHeadings } = this.props;
         return (
             <div className={classes.container}>
-                {
-                    coreSynopsisData.map(item => {
-                        return (
-                            <DashboardCard
-                                id={item.id}
-                                title={item.title}
-                                list={item.list}
-                                loading={loading}
-                                items={item.items}
-                                icon={item.icon}
-                                {...this.props}
-                            />
-                        );
-                    })
-                }
+                <SettingsDialog />
+                <div>
+                    {
+                        synopsisData.map(item => {
+                            return (
+                                <DashboardCard
+                                    showMode={showMode}
```

Comment:
missing `key` prop

Reply:
Key prop was added. I'll try don't forget about it in future.